packages/EasyErrorHandler/src/Bridge/Symfony/Providers/ErrorCodesProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Providers;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use EonX\EasyErrorHandler\Exceptions\ErrorCodesProviderException;
+
+final class ErrorCodesProvider implements ErrorCodesProviderInterface
+{
+    /**
+     * @return array<string, int>
+     *
+     * @throws \EonX\EasyErrorHandler\Exceptions\ErrorCodesProviderException
+     */
+    public function provide(): array
+    {
+        throw new ErrorCodesProviderException('exceptions.error_codes_provider.not_configured');
```

Comment:
The current provider implementation makes no sense I believe. And it makes us think about another way to work with this feature. Let's try the following way:
 - introduce `ErrorCodesInterface` in this package and extend it by the app-level `ErrorCodesInterface`
 - keep `ErrorCodesProviderInterface` and implement `ErrorCodesProvider` with the full providing logic, that we have in Payments now, using the `ErrorCodesInterface` from the point above

That will allow us to have a working default implementation of the provider based on the package-level `ErrorCodesInterface`. All we need to do on the app-level then is only to extend `ErrorCodesInterface`.